app/src/main/java/app/grapheneos/camera/CamConfig.kt

```diff
@@ -1481,6 +1481,20 @@ class CamConfig(private val mActivity: MainActivity) {
         // Add OK and Cancel buttons
         builder.setPositiveButton(getString(R.string.ok)) { _, _ ->
 
+            val commonFormatsEnabled = commonFormats.count {
```

Comment:
`none` would be more efficient than `count` in this case.

Reply:
Yes that's true. I have updated this in the latest changes pushed to the branch. Thanks for reviewing!